packages/modules-sdk/src/loaders/__tests__/medusa-module.ts

```diff
@@ -79,6 +79,29 @@ describe("Medusa Module", () => {
     expect(mockModuleLoader).toBeCalledTimes(2)
   })
 
+  it("MedusaModule bootstrap - Prevent concurrent requests", async () => {
```

Comment:
todo(nit): same remark as for reqwan,  can I let you go through your test and ensure that the tests describe and it are readable. It should looks something along the line of 
```ts
describe("My thing", () => {
  it("should do something like this)
})

```

in this specific case I would expect `Medusa module should prevent concurrent requests to initialize multiple instance of the same module` something along that line. I think we are not consistent across our tests convention and I think it is important for the next person to go through them to read them very clearly and that they are understandable in terms of what we are testing in that context.

wdyt?